src/main/java/Main.java

```diff
@@ -1,5 +1,259 @@
+import java.io.*;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.apache.commons.io.FileUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import static java.lang.System.*;
+
 public class Main {
```

Comment:
Maybe you could've put some of these operations in other classes or you could've created an extra class. There's a lot of different things happening here right now.